trivy 0.26.0 panic scanning local fs while 0.25 works

## Description

<!--
docker fails using trivy:1.26.0 (aka latest) if i reverse to trivy:1.25.0 build passes
-->

## What did you expect to happen?
build should pass and can should work not panic

## What happened instead?
got a #15 6.792 panic: runtime error: invalid memory address or nil pointer dereference

## Output of run with `-debug`:

```
> [builder  9/10] RUN /usr/local/bin/trivy -d fs --exit-code 0 --severity HIGH,CRITICAL --no-progress /:
#15 0.395 2022-04-21T23:28:35.253Z      DEBUG   Severities: HIGH,CRITICAL
#15 0.401 2022-04-21T23:28:35.258Z      DEBUG   cache dir:  /root/.cache/trivy
#15 0.401 2022-04-21T23:28:35.259Z      DEBUG   There is no valid metadata file: unable to open a file: open /root/.cache/trivy/db/metadata.json: no such file or directory
#15 0.401 2022-04-21T23:28:35.259Z      INFO    Need to update DB
#15 0.401 2022-04-21T23:28:35.259Z	INFO	DB Repository: ghcr.io/aquasecurity/trivy-db
#15 0.401 2022-04-21T23:28:35.259Z	INFO	Downloading DB...
#15 0.401 2022-04-21T23:28:35.259Z	DEBUG	no metadata file
#15 6.727 2022-04-21T23:28:41.585Z	DEBUG	Updating database metadata...
#15 6.728 2022-04-21T23:28:41.585Z	DEBUG	DB Schema: 2, UpdatedAt: 2022-04-21 18:07:22.144831392 +0000 UTC, NextUpdate: 2022-04-22 00:07:22.144830992 +0000 UTC, DownloadedAt: 2022-04-21 23:28:41.585399732 +0000 UTC
#15 6.728 2022-04-21T23:28:41.586Z	DEBUG	Vulnerability type:  [os library]
#15 6.792 panic: runtime error: invalid memory address or nil pointer dereference
#15 6.792 [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x16ee9bc]
#15 6.792
#15 6.792 goroutine 365 [running]:
#15 6.792 github.com/aquasecurity/go-dep-parser/pkg/golang/mod.Parse({0x2224da0, 0xc0005c0028})
#15 6.792 	/home/runner/go/pkg/mod/github.com/aquasecurity/go-dep-parser@v0.0.0-20220412145205-d0501f906d90/pkg/golang/mod/parse.go:29 +0x13c
#15 6.793 github.com/aquasecurity/fanal/analyzer/language.Analyze({0x1d16263, 0x5}, {0xc000d11fc1, 0x1e}, {0x2224da0?, 0xc0005c0028?}, 0x51d917?)
#15 6.793 	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220414083417-61bfa9f92483/analyzer/language/analyze.go:16 +0x68
#15 6.793 github.com/aquasecurity/fanal/analyzer/language/golang/mod.gomodAnalyzer.Analyze({}, {0x1cdeac0?, 0x0?}, {{0x7ffe2d606f4b, 0x1}, {0xc000d11fc1, 0x1e}, {0x2235ed8, 0xc0001f85b0}, {0x222fdf8, ...}, ...})
#15 6.794 	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220414083417-61bfa9f92483/analyzer/language/golang/mod/mod.go:39 +0xc5
#15 6.794 github.com/aquasecurity/fanal/analyzer.AnalyzerGroup.AnalyzeFile.func1({0x2231258, 0x31df0a0}, {0x2232c28?, 0xc0005c0028})
#15 6.795 	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220414083417-61bfa9f92483/analyzer/analyzer.go:258 +0x253
#15 6.795 created by github.com/aquasecurity/fanal/analyzer.AnalyzerGroup.AnalyzeFile
#15 6.795 	/home/runner/go/pkg/mod/github.com/aquasecurity/fanal@v0.0.0-20220414083417-61bfa9f92483/analyzer/analyzer.go:253 +0x3d8
------
executor failed running [/bin/sh -c /usr/local/bin/trivy -d fs --exit-code 0 --severity HIGH,CRITICAL --no-progress /]: exit code: 2
make: *** [build] Error 1
```

## Output of `trivy -v`:

```
i am running trivy inside a dockerfile
FROM aquasec/trivy:0.26.0 as trivy
```

## Additional details (base image name, container registry info...):

run trivy this way inside my dockeffile/build

# Run vulnerability scan on build image using latest trivy
WORKDIR /
COPY --from=trivy /usr/local/bin/trivy /usr/local/bin/trivy
RUN /usr/local/bin/trivy -d fs --exit-code 0 --severity HIGH,CRITICAL --no-progress /

--- a/go.mod
+++ b/go.mod
@@ -7,8 +7,8 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/NYTimes/gziphandler v0.0.0-20170623195520-56545f4a5d46
 	github.com/aquasecurity/bolt-fixtures v0.0.0-20200903104109-d34e7f983986
-	github.com/aquasecurity/fanal v0.0.0-20220424145104-2e3e0044128c
-	github.com/aquasecurity/go-dep-parser v0.0.0-20220412145205-d0501f906d90
+	github.com/aquasecurity/fanal v0.0.0-20220425071928-af75d05f1e6e
+	github.com/aquasecurity/go-dep-parser v0.0.0-20220422134844-880747206031
 	github.com/aquasecurity/go-gem-version v0.0.0-20201115065557-8eed6fe000ce
 	github.com/aquasecurity/go-npm-version v0.0.0-20201110091526-0b796d180798
 	github.com/aquasecurity/go-pep440-version v0.0.0-20210121094942-22b2f8951d46
@@ -109,7 +109,7 @@ require (
 	github.com/hashicorp/errwrap v1.1.0 // indirect
 	github.com/hashicorp/go-cleanhttp v0.5.2 // indirect
 	github.com/hashicorp/go-multierror v1.1.1 // indirect
-	github.com/hashicorp/go-retryablehttp v0.7.0 // indirect
+	github.com/hashicorp/go-retryablehttp v0.7.1 // indirect
 	github.com/hashicorp/go-safetemp v1.0.0 // indirect
 	github.com/hashicorp/go-uuid v1.0.2 // indirect
 	github.com/hashicorp/go-version v1.4.0 // indirect

